Automattic/fresh-data
# Remove clientKey

This removes clientKey altogether from the api and associated code. The
clientKey can be directly associated with an instance of the API, we
don't need to keep multiple clients for the same instance. We can just
create multiple instances of the same API.

To Test:
1. `npm install`, `npm test` and ensure all tests pass.

For further cumulative testing of this and associated PRs, visit #81
